### PR TITLE
Create BasicJava Project

### PR DESCRIPTION
**Pull request by ngannt24**

### CODE REVIEW

File: src/com/gmo/ngannt/loops.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			 for(int Count = 0;Count<=10;Count++){
+				 if(Count==6){
+				 break;
+				 }
+				 System.out.println("Count is ==> " + Count );
+				 }
+				 System.out.println("You have exited the loop");
+				 
+			//The continue Keyword
+				 for(int Count = 0;Count<=5;Count++){
```
Comment (TuyenLMQC) [inline src/com/gmo/ngannt/loops.java]: Format code nhé

File: src/com/gmo/ngannt/operators.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	static int speed = 80;
+	 static int distance = 20;
+	 static int time = 10;
+	 static String name = "ToolsQA";
+	 static boolean isGood = true;
+	 
+	public static void main(String[] args) {
+
+		System.out.println("Value stored in the speed variable is : " + speed);
+		 System.out.println("Value stored in the distance variable is : " + distance);
```
Comment (TuyenLMQC) [inline src/com/gmo/ngannt/operators.java]: Format code nhé

File: src/com/gmo/ngannt/loops.java (hunk 1 above)
Comment (ngannt24) [reply src/com/gmo/ngannt/loops.java]: fixed

File: src/com/gmo/ngannt/operators.java (hunk 2 above)
Comment (ngannt24) [reply src/com/gmo/ngannt/operators.java]: fixed

**Pull request merged**